.circleci/config.yml

```diff
@@ -598,16 +598,19 @@ jobs:
       - run:
           name: Generate diff
           command: |
-            diff -bur --no-dereference \
+            diff -bur --no-dereference --new-file \
```

Comment:
One edge case that `--new-file` would now obscure is empty files being added/removed.

Reply:
To solve that we could add another pair of `diff` commands for DAGs and SQL (before the corresponding existing `diff` command) like `diff -qr ... | grep '^Only in'` to still list files that only exist in one place or the other.

Reply:
#4600